app/src/main/res/values/strings.xml

```diff
@@ -12,5 +12,30 @@
     <string name="page_2_toolbar_title">Example Request</string>
     <string name="fragment_example_username">Username</string>
     <string name="fragment_example_login">Login</string>
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="terms">By joining you agree to our <u>Terms &amp; Conditions</u></string>
+    <string name="hello_blank_fragment">Hello blank fragment</string>
+    <string name="email">Email</string>
+    <string name="password">Password</string>
+    <string name="key_email">key_email</string>
+    <string name="key_password">key_password</string>
+    <string name="key_login">key_login</string>
+    <string name="mandatory_fields">All fields are mandatory</string>
+    <string name="example_email">Incorrect email format. A correct format is \'example@domain.com\'</string>
+    <string name="log_in">LOG IN</string>
+    <string name="sign_up">SIGN UP</string>
+    <string name="home_page">Home Page</string>
+    <string name="incorrect_email_password">The Email address or Password is incorrect</string>
+    <string name="call_failure">Call Failure</string>
+    <string name="no_internet">No internet connection</string>
+    <string name="wolox">Wolox</string>
+    <string name="profile_fragment">Profile Fragment</string>
+    <string name="title">Title</string>
+    <string name="description">Description</string>
+    <string name="_15_min">15 min</string>
+    <string name="no_news">No more news to show</string>
+    <string name="profile">Profile</string>
+    <string name="news_fragment">News Fragment</string>
+    <string name="news">News</string>
```

Comment:
I've made the same comment in other PRs, use more descriptive ids for strings, dimens, etc. For example, If the app is getting bigger I wouldn't know where no_news is called

Reply:
What happened with this?